Source/KakapoDB.swift

```diff
@@ -157,15 +157,17 @@ public final class KakapoDB {
     /**
      Deletes the given Storable object
      
-     - parameter entity: The Storable object to be deleted
+     - parameter entity: The Storable object to be deleted. If an object with the same id and type can be found then will be deleted, even if the objects are not really equal.
```

Comment:
I would rephrase this: `The Storable object to be deleted: the database will delete any object found with same id and type`